webui/react/src/contexts/Store.tsx

```diff
@@ -60,6 +63,7 @@ type Action =
 | { type: StoreAction.SetAuth; value: Auth }
 | { type: StoreAction.SetAuthCheck }
 | { type: StoreAction.SetInfo; value: DeterminedInfo }
+| { type: StoreAction.SetNotebookConfig; value: NotebookConfig }
```

Comment:
**question:** looks like this isn't really being used? maybe old code that just needs to be pruned? same with the store action in `NotebookModalContainer` for the NotebookModal story

Reply:
basically the notebook config state can probably persist at the modal component level and not have to be accessible or shared beyond that boundary

Reply:
Storybook yells at me that "useStore must be used within a StoreProvider" if I don't do this, so I added it.

Reply:
The tickets call for form options except for name to be saved in local storage, I can take that out if you'd prefer though.